docs/library/spi_engine/instruction-format.rst

```diff
@@ -69,7 +69,7 @@ instruction is twice the delay.
 
```

Comment:
Can you also add information about the minimum delay necessary to grab a new instruction?
Like, when [9:8] == 0, there is still 1 or 2 clock-cycles spent.